inductiva/coastal/README.md

```diff
@@ -8,10 +8,76 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 
 ### Example
 
-Start by creating or loading a bathymetry. This can be done e.g. by reading an
+Note: You can follow this example in a notebook or run the script
+`random_bathymetry_demo.py` available in this folder, that contains all the code
```

Comment:
I still don't think this should live inside the coastal module. Move it to the demos.